tests/foreman/cli/test_ldapauthsource.py

```diff
@@ -482,7 +486,7 @@ def test_rhsso_login_using_hammer(
 
         :CaseImportance: High
         """
-        result = AuthLogin.oauth(
+        result = destructive_sat.cli.AuthLogin.oauth(
             {
```

Comment:
@JacobCallahan I get that this will be executed on destructive satellite created using `destructive_sat` but what about the code written in fixtures that this test is using(ex. rhsso_setting_setup)? Won't we need to change that as well, like you have done here.

Reply:
the destructive_sat fixture uses the Satellite object's context manager to force robottelo to point to the destructive satellite while the test is running.